notebooks/tabular_DL_regression.ipynb

```diff
@@ -0,0 +1,283 @@
+    }
+   },
+   "outputs": [],
+   "source": [
+    "def baseline_model(input_dim):\n",
+    "\tmodel = Sequential()\n",
+    "\tmodel.add(Dense(13, input_dim=input_dim, kernel_initializer='normal', activation='relu'))\n",
+    "\tmodel.add(Dense(1, kernel_initializer='normal'))\n",
+    "\tmodel.compile(loss='mean_squared_error', optimizer='adam')\n",
+    "\treturn model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "python"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "model = baseline_model(X_train.shape[1])\n",
+    "model.fit(X_train, y_train, epochs=5, batch_size=10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "python"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "y_pred_rnn = model.predict(X_test) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "python"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "mae_rnn = mean_absolute_error(y_test, y_pred_rnn)\n",
+    "mse_rnn = mean_squared_error(y_test, y_pred_rnn)\n",
+    "r2_rnn = r2_score(y_test, y_pred_rnn)\n",
+    "\n",
+    "print(\"MAE : \", mae_rnn)\n",
+    "print(\"MSE : \", mse_rnn)\n",
+    "print(\"R2 : \", r2_rnn)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use intelligibility from readml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    " **WARNINGS**: Take care of change de config_local.cfg to adapt it to the use case, you may need to re run ```pip install -e .``` after the change in the configuration."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "python"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "model_explain = model # The model you use\n",
+    "task = \"regression\" # here we try to solve a regression problem\n",
+    "features_name = list(X_train.columns) # all the features without de target column\n",
+    "out_path = \"../outputs/notebooks/\" # the path where you want to save the report\n",
+    "res_dir = os.path.join(ROOT_DIR, out_path)\n",
+    "if not os.path.exists(res_dir)\n",
+    "    os.makedirs(res_dir)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    " If you just want to try it, take care of take only the ```.head()``` to have report on only a small part of the observation, because this would run and do intelligibility on each row you give to the ```.explain_tabular()``` method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "python"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "exp = ExplainDL(\n",
+    "                model=model_explain,\n",
+    "                out_path=out_path,\n",
```

Comment:
I guess you should use 

out_path=res_dir 

instead of 

out_path=out_path